Tests/Test_Renode_MotorSimulator.c

```diff
@@ -0,0 +1,163 @@
+#include "os.h"
+#include "CANbus.h"
+#include <stdio.h>
+#include "Tasks.h"
+#include <math.h>
+
+// Global variables
+static float setpointVelocity = 0.0f; 
+static float velocity = 0.0f;
+static float current = 0.0f; 
+static float total_accel = 0.0f;
+static CANDATA_t oldCD;
+static uint8_t cycle_ctr = 0;
+static uint16_t motor_force = 0;
+static uint8_t forceLUTIndex = 0;
+
+// FORCE LUT 
+// FORCELUT Size
+#define FORCELUT_SIZE 21 
+// Force Lookup Table: Indexed by Current%toIndex, returns a force applied by current % (in Newtons)
+static const int FORCELUT[FORCELUT_SIZE] = {
+    0,
+    309,
+    618,
+    926,
+    1235,
+    1544,
+    1853,
+    2162,
+    2471,
+    2779,
+    3088,
+    3397,
+    3706,
+    4015,
+    4324,
+    4632,
+    4941,
+    5250,
+    5559,
+    5868,
+    6176
+};
+
+// Macros for calculating the velocity of the car
+#define MS_TIME_DELAY_MS 100
+#define DECELERATION 2.0 // In m/s^2
```

Comment:
If last year's car is anything to go by, the deceleration when coasting is less than this. Not sure if it'll make that big of a difference in the simulations though.